docs/content/preview/drivers-orms/include-drivers-orms-list.md

```diff
@@ -117,6 +117,8 @@ private = true
 
 | ORM        | Version | Support Level | Example apps |
 | :--------- | :------ | :------------ | :----------- |
+| **Driver** | | | |
+| Rust-postgres Driver |         | Full | [CRUD](/preview/drivers-orms/rust/yb-rust-postgres/) |
```

Comment:
@HarshDaryani896 - can you confirm the version ? Couldn’t find it.

Reply:
We have 2 rust drivers:
[yb-postgres](https://crates.io/crates/yb-postgres) (synchronous YSQL client): v0.19.7-yb-1-beta
[yb-tokio-postgres](https://crates.io/crates/yb-tokio-postgres) (asynchronous YSQL client): v0.7.10-yb-1-beta